pkgs/shells/fish/babelfish.nix

```diff
@@ -17,5 +17,6 @@ buildGoModule rec {
     homepage = "https://github.com/bouk/babelfish";
     license = licenses.mit;
     maintainers = with maintainers; [ bouk kevingriffin ];
+    platforms = platforms.unix;
```

Comment:
This is set by buildGoModule.

Reply:
in master:
```
nix-repl> babelfish.meta.platforms
[ "aarch64-linux" "armv5tel-linux" "armv6l-linux" "armv7a-linux" "armv7l-linux" "mipsel-linux" "i686-linux" "x86_64-linux" "powerpc64le-linux" "riscv32-linux" "riscv64-linux" "x86_64-darwin" "i686-darwin" "aarch64-darwin" "armv7a-darwin" ]
```

Reply:
I totally misread the `buildGoModule`'s `platforms` as being just `lib.platforms.all`, not `go.meta.platforms or lib.platforms.all` (which is basically just `platforms.linux ++ platforms.darwin`). I'd still prefer explicit platforms being set in Nixpkgs, but this is a discussion / RFC / what have you for another day. Feel free to close (or merge).